bigtable/quickstart/main.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2018 Google LLC
```

Comment:
Would you mind copying the license header from the other files? The linter is too strict (needs to be updated).

Reply:
done. you might want to fix the license check in the linter though to reflect the latest expected style of license.